### PR TITLE
build(update): ruby v3

### PR DESCRIPTION
**Pull request by roschaefer**
This should update only ruby to v3. I specified `alpine3.12` because
`alpine3.13` already has just postgres v13 tooling in their package
registry.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,8 +7,8 @@ RUN apk --update add \
     git \
     nodejs \
     yarn \
-    postgresql-dev=~12.6 \
-    postgresql-client=~12.6 \
+    postgresql-dev=~13.2 \
```
[the remark this answers is not in the document]
Comment (tillprochaska) [reply Dockerfile:10]: @roschaefer I think we’d still want to upgrade to Ruby 3, right?

File: app/components/attribute_bag.rb [hunk 2]
```diff
@@ -30,7 +30,7 @@ def merge(**additional_attrs)
     self.class.new(**new_attrs.deep_merge(additional_attrs))
   end
 
-  def defaults(**default_attrs)
+  def defaults(default_attrs = {})
```
Comment (roschaefer) [inline app/components/attribute_bag.rb:33]: @tillprochaska there is more in this file, I'm talking about `slice` and `except`. Apparently it's not executed during tests, can we remove?

Comment (tillprochaska) [reply app/components/attribute_bag.rb:33]: @roschaefer 

Both are used in components and tested:

https://github.com/tactilenews/100eyes/search?q=slice
https://github.com/tactilenews/100eyes/search?q=except

File: config/initializers/new_framework_defaults_6_1.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
```
Comment (tillprochaska) [inline config/initializers/new_framework_defaults_6_1.rb:1]: Why is this file added in this PR? We are already on Rails 6.1…

File: app/components/attribute_bag.rb (hunk 2 above)
Comment (roschaefer) [reply app/components/attribute_bag.rb:33]: Right! I oversaw they have only one asterisk. So let's merge?

**Review by tillprochaska — APPROVED** [review]
